### DISCUSSION TITLE
prevent summary of class to be repeated for all members without own

### DISCUSSION THREAD

**Discussion by Stoffelche** [dotnet/docfx#8383] [Q&A] [answered]
It seems that the summary of a class is repeated for all members (properties, methods, fields) that don't have own summary. 
This is confusing, especially when the class description is somewhat longer.

How can this behavior be disabled? The summary of a type should not be repeated for each member.
TIA, for any hint.
Stefan

**Comment by KalleOlaviNiemitalo** [comment, marked as answer]
That was also reported as <https://github.com/dotnet/docfx/issues/8369> and fixed in DocFX v2.61.0.

**Reply by Stoffelche** [reply]
Thank you very much for your assistance. Works with 2.6.1!
